os/fs/smartfs/smartfs.h

```diff
@@ -416,6 +416,8 @@ uint32_t smartfs_rdle32(FAR const void *val);
 
 void smartfs_wrle32(uint8_t *dest, uint32_t val);
 
+bool is_path_last_slash(const char* path);
```

Comment:
How about changing the function name from "is_path_last_slash" to "path_ends_with_slash" ?